sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py

```diff
@@ -854,7 +855,7 @@ def create_issuer(self, name, provider, **kwargs):
             issuer_attributes = None
         issuer_bundle = self._client.set_certificate_issuer(
             vault_base_url=self.vault_url,
-            issuer_name=name,
+            issuer_name=name.value if not isinstance(name, str) else name,
```

Comment:
The serializer should handle this for you (try it).

Reply:
nice, got rid of these checks in the clients